actions/check_if_member_MOD.js

```diff
@@ -178,10 +178,10 @@ module.exports = {
         result = Boolean(member.voice?.deaf);
         break;
       case 9:
-        result = member.id === msg.author.id;
+        result = member.id === msg?.author.id || member.id === interaction.member.id;
```

Comment:
```suggestion
        result = member.id === msg?.author.id ?? interaction.member.id;
```

Reply:
Parenthesis are required here because of the order of the conditions, and we can use `interaction.user` here. I've edited the suggestion to reflect this

Reply:
Awesome thanks bud

Reply:
:)